types/fetch-mock/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "strict-export-declare-modifiers": false
```

Comment:
Thanks, @andy-ms, this removed the issues for me as well! Thus I reverted the change in `tslint.json`.